content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_design_dialog/.content.xml

```diff
@@ -112,6 +112,12 @@
                                     jcr:primaryType="nt:unstructured"
                                     sling:resourceType="granite/ui/components/foundation/form/autocomplete/list"/>
                             </appResourcesClientlib>
+                            <mainContentSelector
+                                jcr:primaryType="nt:unstructured"
+                                fieldLabel="Skip to main content element selector"
+                                fieldDescription="This property defines selector wchich links to main part of the page. It is used by accessibility feature which allows user to skip directly to the main content of the page."
```

Comment:
I'd rephrase as: 
Selector for the main section of the page. Used as accessibility feature to skip directly to the main content of the page.